charts/ingress-nginx/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -33,7 +33,7 @@ spec:
           imagePullPolicy: {{ .Values.controller.admissionWebhooks.patch.image.pullPolicy }}
           args:
             - create
-            - --host={{ template "ingress-nginx.controller.fullname" . }}-admission,{{ template "ingress-nginx.controller.fullname" . }}-admission.{{ .Release.Namespace }}.svc
+            - --host={{ include "ingress-nginx.fullname" . }}-controller-admission,{{ include "ingress-nginx.fullname" . }}-controller-admission.{{ .Release.Namespace }}.svc
```

Comment:
I'm wondering why you are switching from template to include. The only difference between the two is that include can be used with other template functions. Generally, I avoid using include unless I am using some function against the return.

Reply:
It's what `helm create` does: https://github.com/helm/helm/blob/16024dc19a23e83f00a19742033031717a56be0e/pkg/chartutil/create.go#L230

Reply:
I didn't know that. In another pr lets just switch everything over to include then.

Reply:
I believe it's fine to switch template to include in this one. https://github.com/helm/charts/issues/3011#issuecomment-457640417